doc/tutorials/nires_howto.rst

```diff
@@ -0,0 +1,435 @@
+
+.. include:: ../include/links.rst
+
+.. _nires_howto:
+
+================
+Keck/NIRES HOWTO
+================
+
+Overview
+========
+
+This doc goes through a full run of PypeIt on one of the Keck/NIRES datasets
+(specifically the ``NIRES`` dataset) in the `PypeIt Development Suite`_ (see
+:ref:`here <dev-suite>`).
+
+----
+
+Setup
+=====
+
+Before reducing your data with PypeIt, you first have to prepare a
+:ref:`pypeit_file` by running :ref:`pypeit_setup`:
+
+.. code-block:: bash
+
+    pypeit_setup -r $HOME/Work/packages/PypeIt-development-suite/RAW_DATA/keck_nires/NIRES/ -s keck_nires -b -c A 
+
+where the ``-r`` argument should be replaced by your local directory and the
+``-b`` indicates that the data uses background images and should include the
+``calib``, ``comb_id``, ``bkg_id`` in the pypeit file.  This directory only has
+one instrument configuration (NIRES only has one configuration anyway), so
+setting ``-c A`` and ``-c all`` is equivalent.
+
+This will make a directory called ``keck_nires_A`` that holds a pypeit file
+called ``keck_nires_A.pypeit`` that looks like this:
+
+.. include:: ../include/keck_nires_A.pypeit.rst
+
+For this example dataset, the details of the default pypeit file are not
+correct; see :ref:`here <nires_config_report>` for an example where the frame
+typing and dither pattern determinations *are* correct by default.
+
+The issue with this dataset is that only two pointings in the ABBA pattern for
+the standard star observations are available, the exposure time of the standard
+observations is longer than expected, and the dither pattern of the science
+frames are set to MANUAL.  All of this means you need to edit the pypeit file to
+correct specific errors.  The corrections needed are:
+
+    - Set the frametypes for ``s190519_0059.fits`` and ``s190519_0060.fits`` to
+      ``arc,standard,tilt`` (i.e., change ``science`` to ``standard`` for these
+      frames).
+
+    - Set the ``bkg_id`` for the ``standard`` and ``science`` frames; see below
+      and :ref:`a-b_differencing`.
+
+The corrected version looks like this (pulled directly from the :ref:`dev-suite`):
+
+.. include:: ../include/keck_nires_A_corrected.pypeit.rst
+
+Use of the standard frame
+-------------------------
+
+By setting the science and standard frames to also be ``arc`` and ``tilt``
+frames, we're indicating that the OH sky lines should be used to perform the
+wavelength calibration.  The wavelength calibration for these two sets of frames
+will be performed *independently* because we assign them to different
```

Comment:
We should co-add them, actually. This is advised in all cases in NIR to dilute the contribution from the object flux.

Reply:
Even if they have different exposure times?  For the time being, I've added some comments that all 4 frames can be combined so that there is only one calibration group.  I've also added a TODO comment that we should clean this up.  We likely want the tutorial to focus on a very clean dataset where everything is straight-forward and then provide come caveats either in this tutorial or in a dedicated NIRES page.